main.tf

```
@@ -31,3 +32,11 @@ module "cloud_provider_equinix_metal" {
   addon_config  = var.cloud_provider_equinix_metal_config
   addon_context = local.addon_context
 }
+
+module "portworx" {
+  count  = var.enable_portworx ? 1 : 0
+  source = "./modules/portworx"
+
+  kubeconfig_local_path = var.kubeconfig_local_path
+  portworx_config       = var.portworx_config
```

Let's use `addon_context` instead, like we do with kubeconfig_remote_path. 

```suggestion
  addon_context   = local.addon_context
  portworx_config = var.portworx_config
```